app/frontend/src/internationalization/trans/chefs/en/en.json

```diff
@@ -199,7 +199,10 @@
     "eventSubscription": "Event Subscription",
     "validEndpointRequired": "Please enter a valid endpoint starting with https://",
     "validBearerTokenRequired": "Enter a valid bearer token example: 89abddfb-2cff-4fda-83e6-13221f0c3d4f",
-    "wideFormLayout": "Wide Form Layout"
+    "wideFormLayout": "Wide Form Layout",
+    "formMetadataTitle": "Form Metadata",
```

Comment:
For accessibility, we do need translations for all languages for any added text.

Reply:
added a ticket, will address this.

Reply:
done for all languages.